priv/gettext/fr/LC_MESSAGES/privacy.po

```diff
@@ -14,175 +14,175 @@ msgstr ""
-msgstr ""
+msgstr "Divulgation de vos informations"
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:2
 msgid "English version of the Privacy Policy"
-msgstr ""
+msgstr "Version anglaise de la politique de confidentialité"
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:13
 msgid "How cookies are used"
-msgstr ""
+msgstr "Comment les cookies sont utilisés"
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:41
 msgid "If Listudy is sold, the data provided by the user (such as emails, pgns, comments) will be transferred to a new third party owner."
-msgstr ""
+msgstr "Si Listudy est vendu, les données fournies par l'utilisateur (telles que les e-mails, les pages Web, les commentaires) seront transférées à un nouveau propriétaire tiers."
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:24
 msgid "If you create an account, you will be asked to provide a email address. This email address does not have to be a real email address."
-msgstr ""
+msgstr "Si vous créez un compte, il vous sera demandé de fournir une adresse e-mail. Cette adresse e-mail ne doit pas nécessairement être une véritable adresse e-mail."
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:21
 msgid "Information that Listudy collects:"
-msgstr ""
+msgstr "Informations que Listudiy collecte :"
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:48
 msgid "Information you delete is deleted from the live site but remains in Listudy’ backups for up to one year."
-msgstr ""
+msgstr "Les informations que vous supprimez sont supprimées du site en ligne mais restent dans les sauvegardes de Listudy pendant un an."
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:6
 msgid "Introduction"
-msgstr ""
+msgstr "Introduction"
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:4
 msgid "Last edited:"
-msgstr ""
+msgstr "Dernière modification:"
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:60
 msgid "Listudy can change this policy at its discretion."
-msgstr ""
+msgstr "Listudiy peut modifier cette politique à sa discrétion."
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:7
 msgid "Listudy.org (“Listudy”) is an online chess site. Users can optionally register for an account; however, certain features are available for registered users only. This is because Listudy needs basic information from its users in order to function."
-msgstr ""
+msgstr "Listudy.org (“Listudy”) est un site d'échecs en ligne. Les utilisateurs peuvent éventuellement créer un compte ; cependant, certaines fonctionnalités ne sont disponibles que pour les utilisateurs enregistrés. En effet, Listudy a besoin d'informations de base de ses utilisateurs pour fonctionner."
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:14
 msgid "Managing you information"
```

Comment:
Is this a typo or is it on purpose?

Reply:
Do you mean `Managing your information`? It is a sub header on this page https://listudy.org/en/privacy ; I don't see a typo?

Reply:
Yes, it seems like an `r` is missing?
(The typo is not in the sub header, but in the detail list)
```suggestion
msgid "Managing your information"
```

Reply:
Oh im blind. Again it should get fixed in code not in the po files.